sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -163,6 +163,10 @@ abstract class Scrollable extends StatefulComponent {
     return scrollTo(newScrollOffset, duration: duration, curve: curve);
   }
 
+  void fling(Offset velocity) {
+    _handleDragEnd(velocity);
+  }
```

Comment:
I almost did that but saw another class had a function called _handleDragEnd and thought maybe it was overriding it.